src/core_modules/capture-core/components/D2Form/field/validators/getValidators.js

```diff
@@ -212,14 +212,14 @@ function buildTypeValidators(metaData: DataElement | DateDataElement): ?Array<Va
 
     validatorContainersForType = validatorContainersForType.map(validatorContainer => ({
         ...validatorContainer,
-        validator: (value: any) => {
+        validator: (value: any, validationContext: ?Object, internalError: ?Object) => {
```

Comment:
We should end up having `commitOptions` (containing the internal error) as the third param here instead of `internalError`.